### ISSUE TITLE
Rollbar 2.19+ Incompatible with earlier Capistrano version

### ISSUE THREAD

**Issue by mehagar** [rollbar/rollbar-gem#975]
I am using version `2.14.2` of the `capistrano` gem with `rollbar` version `2.26.0`. When I run the `cap rollbar:deploy_started`, I get this error:

```
/Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/rollbar-2.26.0/lib/rollbar/capistrano_tasks.rb:64:in `capistrano_300_warning': uninitialized constant Capistrano::VERSION (NameError)
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/rollbar-2.26.0/lib/rollbar/capistrano_tasks.rb:40:in `deploy_task'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/rollbar-2.26.0/lib/rollbar/capistrano_tasks.rb:8:in `deploy_started'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/rollbar-2.26.0/lib/rollbar/capistrano.rb:54:in `block in load_deploy_started'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/rollbar-2.26.0/lib/rollbar/capistrano.rb:73:in `block (3 levels) in load_task'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/configuration/execution.rb:138:in `instance_eval'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/configuration/execution.rb:138:in `invoke_task_directly'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/configuration/callbacks.rb:25:in `invoke_task_directly_with_callbacks'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/configuration/execution.rb:89:in `execute_task'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/configuration/execution.rb:101:in `find_and_execute_task'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/cli/execute.rb:46:in `block in execute_requested_actions'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/cli/execute.rb:45:in `each'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/cli/execute.rb:45:in `execute_requested_actions'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/cli/help.rb:19:in `execute_requested_actions_with_help'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/cli/execute.rb:34:in `execute!'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/lib/capistrano/cli/execute.rb:14:in `execute'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/gems/capistrano-2.14.2/bin/cap:4:in `<top (required)>'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/bin/cap:23:in `load'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/bin/cap:23:in `<main>'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/bin/ruby_executable_hooks:24:in `eval'
	from /Users/michaelhagar/.rvm/gems/ruby-2.1.10@sa/bin/ruby_executable_hooks:24:in `<main>'
```

This is an issue on rollbar `2.19.0` and newer. Previous version of rollbar work fine.

**Comment by waltjones** [comment]
@mehagar I see what's happening. Capistrano 2.x requires using `Capistrano::Version.to_s`.

A fix can go in the next rollbar-gem release.

**Comment by waltjones** [comment]
Fix released in v2.26.1 https://github.com/rollbar/rollbar-gem/releases/tag/v2.26.1
